src/Catty/ViewController/Upload/LoginViewController.m

```diff
@@ -367,17 +369,22 @@ -(void)handleLoginResponseWithData:(NSData *)data andResponse:(NSURLResponse *)r
         self.loginButton.enabled = YES;
         [self hideLoadingView];
         NSDebug(@"Error: %@", kLocalizedAuthenticationFailed);
-        [Util alertWithText:kLocalizedAuthenticationFailed];
+        [self showError:kLocalizedAuthenticationFailed];
     } else {
         self.loginButton.enabled = YES;
         [self hideLoadingView];
         
         NSString *serverResponse = [dictionary valueForKey:answerTag];
         NSDebug(@"Error: %@", serverResponse);
-        [Util alertWithText:serverResponse];
+        [self showError:kLocalizedServerTimeoutIssueMessage];
     }
 }
 
+-(void)showError:(NSString *)message {
+    [Util alertWithText:message];
+    self.errorMessage = message;
```

Comment:
Why do you need this property here? Don't you already have it in the Mock?

Reply:
You are right. I removed the property from the project and rebased and squashed again.